.github/CONTRIBUTING.md

```diff
@@ -99,6 +99,39 @@ The previous code made compliant:
 	code
 ```
 
+### All process procs need to make use of delta-time and be frame independent
+
+In a lot of our older code, process() is frame dependent. As an example, here's some example mob code.
+
+```DM
+/mob/testmob
+	var/health = 100
+	var/health_loss = 4 //We want to lose 2 health per second, so 4 per SSmobs process
+
+/mob/testmob/process(delta_time) //SSmobs runs once every 2 seconds
```

Comment:
Isn't this meant to be the one with our delta_time

Reply:
it's a default arg on process(), It's just not being utilized in this example.